packages/launchpad/cypress/e2e/project-setup.cy.ts

```diff
@@ -305,6 +307,7 @@ describe('Launchpad: Setup Project', () => {
           cy.containsPath('cypress/support/e2e.js')
           cy.containsPath('cypress/support/commands.js')
           cy.containsPath('cypress/fixtures/example.json')
+          cy.get('[data-cy=collapsible-header]').should('have.attr', 'aria-expanded', 'false')
```

Comment:
I don't think this is fully doing what you might expect:

![image](https://user-images.githubusercontent.com/19196536/188055617-93fd90eb-9572-4f37-94f2-a9eadf85c2a9.png)

As long as **one** is `false`, this will pass. I've got a valid one open here, and it isn't failing the test. 

We might want to add some extra data selectors, so something like `data-cy=valid`, than assert all those are collapsed. It might look like

```js
cy.findByTestId('valid').should('have.attr', 'aria-expanded', 'false')
```

Reply:
This was a great catch! As you mentioned, the `cy.get('[data-cy=collapsible-header]').should('have.attr', 'aria-expanded', 'false')` would pass as long as at least 1 collapsible-header was closed which does not test all of them being closed.

After some troubleshooting and with the help of Mark we came up with a solution:
`cy.get('[data-cy=valid] [data-cy=collapsible-header]').each((element) => {
          cy.wrap(element).should('have.attr', 'aria-expanded', 'false')
        })`
This individually asserts the collapsible headers of a certain type. I made the updates to all the tests to reflect this change